resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -256,7 +256,7 @@ private[spark] object Config extends Logging {
   private val podConfValidator = (s"^$dns1123LabelFmt(\\.$dns1123LabelFmt)*$$").r.pattern
 
   // The possible longest executor name would be "$prefix-exec-${Int.MaxValue}"
-  private def isValidExecutorPodNamePrefix(prefix: String): Boolean = {
+  def isValidExecutorPodNamePrefix(prefix: String): Boolean = {
```

Comment:
In this case, we may want move this to `KubernetesUtils` officially.

Reply:
sounds godd, moved this!